### ISSUE TITLE
Mario Forever Flash: Stuck at game loaded menu.

### ISSUE THREAD

**Issue by p0008874** [ruffle-rs/ruffle#9534]
### Describe the bug

https://user-images.githubusercontent.com/75534590/218322788-2d2157fd-0585-4f28-8898-710b97e03846.mp4

```
2023-02-12T16:10:54.455329Z  WARN run_frame:run_all_phases_avm2: ruffle_core::stub: Encountered stub: AVM2 flash.system.Security.allowDomain getter
2023-02-12T16:10:54.457434Z  WARN run_frame:run_all_phases_avm2: ruffle_core::stub: Encountered stub: AVM2 flash.system.Security.allowInsecureDomain getter
2023-02-12T16:10:54.459540Z  WARN run_frame:run_all_phases_avm2: ruffle_core::stub: Encountered stub: AVM2 flash.net.LocalConnection.allowDomain()
2023-02-12T16:10:54.460050Z  WARN run_frame:run_all_phases_avm2: ruffle_core::stub: Encountered stub: AVM2 flash.net.LocalConnection.allowInsecureDomain()
2023-02-12T16:10:54.460515Z  WARN run_frame:run_all_phases_avm2: ruffle_core::stub: Encountered stub: AVM2 flash.net.LocalConnection.connect()
2023-02-12T16:10:55.001098Z ERROR ruffle_core::loader: Error during movie loading: FetchError("HTTP status is not ok, got 404 Not Found")
2023-02-12T16:10:57.468427Z  INFO run_frame:run_all_phases_avm2: avm_trace: [MochiAd] Couldn't load an ad, make sure your game's local security sandbox is configured for Access Network Only and that you are not using ad blocking software
2023-02-12T16:10:57.735374Z  WARN ruffle_core::stub: Encountered stub: AVM2 flash.net.LocalConnection.close()
```

### Expected behavior

https://user-images.githubusercontent.com/75534590/218322805-d8d281fc-d930-413f-ac9a-f38c3b4bf7d9.mp4

### Affected platform

Desktop app

### Operating system

Windows 10

### Additional information

[marioforeverflash.zip](https://github.com/ruffle-rs/ruffle/files/10716543/marioforeverflash.zip)

**Comment by n0samu** [comment]
This seems like another mouse pick issue. The game's onEnterFrame handler has this:
`gfx.addEventListener(MouseEvent.CLICK,clickman2);`
I can confirm that this code runs. And here is the clickman2 function:
```as
public function clickman2(param1:MouseEvent) : void
{
    gfx.removeEventListener(MouseEvent.CLICK,clickman2);
    removeChild(gfx);
    nextFrame();
    init();
}
```
This function never runs. So the `gfx` sprite is somehow never receiving the mouse event.

**Comment by n0samu** [comment]
Fixed by #9867
